pkg/pods/pod.go

```diff
@@ -186,10 +186,6 @@ func (pod *Pod) buildRunitServices(launchables []hoist.Launchable) error {
 		for _, executable := range executables {
 			sbTemplate.AddEntry(executable.Service.Name, executable.Exec)
 		}
```

Comment:
Removing this error message seems unrelated to the TLS configuration change. Can you put it in its own commit?